docs/howto/blueprints.rst

```diff
@@ -0,0 +1,24 @@
+Use Blueprints
+--------------
+
+You can use blueprints to break up your tasks.
+
+Firstly use blueprints to create a task collection::
```

Comment:
```suggestion
First, use blueprints to create a task collection::
```

Reply:
I took a second stab at this HowTo.  Please re-review.